### DISCUSSION TITLE
include html in index.html

### DISCUSSION THREAD

**Discussion by wlwatkins** [wailsapp/wails#1687] [Q&A]
Is there a way to include html within the index.html. 
Indeed, I don't believe wail supports multi windows in windows. So I have to put everything within the one window. 
I have tabs at the top for each main functionality. 
My idea is to have the main index.html file with the skeleton of the program and within it include a separate html file for each sub window corresponding to each tab. hence preventing a supper long index.html. 

is this possible?

**Comment by hanafiqp** [comment]
Hi @willmendil , have you try this tutorials ? 

https://www.w3schools.com/howto/howto_html_include.asp

**Reply by wlwatkins** [reply]
I ended learning svelte ;)

**Reply by leaanthony** [reply]
Good choice!!!
